### ISSUE TITLE
Images are being served via imgur which has a restrictive CORS policy

### ISSUE THREAD

**Issue by lloydjatkinson** [r-spacex/SpaceX-API#1189]
Hi,

Would it be possible to switch to, for example, imgbox, as shown in the README example anyway? The reason for this is that imgur now does not allow embedding of images without an API key. imgbox does not.

Thanks!

**Comment by jakewmeyer** [comment]
Definitely! I'll throw a script together this weekend to get them all moved over 👍

**Comment by jakewmeyer** [comment]
@lloydjatkinson Just switched everything over, should be good now 👍

**Comment by lloydjatkinson** [comment]
That's great thank you! I can still see 4 from this API, but its better than the hundreds previous 😁

![image](https://user-images.githubusercontent.com/5633938/187052044-639a9f64-4925-44a6-bf3a-aff85e8d5e27.png)

**Comment by jakewmeyer** [comment]
Might been a few lingering cache items at that point, all imgur links appear to be gone now
